transcom/mymove
# [MB-3589] Pass move order duty station postal code when there is no destination address

## Description

A previous PR allowed for the display of an address object with only the postal code present but didn't actually allow for a destination address to not exist.

This PR will use the new duty station postal code, which is included in the move order record, in lieu of a missing destination address on a shipment.

## Reviewer Notes

There's not a ton to review, mostly the orders info object we are already building from the the move order response is now passed to the RequestedShipment component.  The destination address gets displayed on the move details page when there are shipments to be approved and then again on the preview modal before submitting the mto.

**Stories were edited but there are no visual changes to inspect, the component just needed duty station data passed to it** 

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make client_run
```

To test:

1. Create a new milmove service member account at milmovelocal:3000
2. Create a new HHG move without a destination address, note the new duty station postal code
3. Login as a TOO, find the unapproved move at the end of the list
4. Follow the customer details link to see the destination address is empty
5. Go to the Move Details page to view the shipment needing approval shows the duty station postal code
6. Select the shipment and approve the service items to reveal the approval modal
7. The duty station postal code again should be displayed in the destination address field

## Code Review Verification Steps

 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/backend#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3589) for this change

## Screenshots

![Screen Shot 2020-08-10 at 11 32 50 AM](https://user-images.githubusercontent.com/52669884/89801530-50c72c00-dafe-11ea-9087-77c7a98f5a9e.png)
![Screen Shot 2020-08-10 at 11 33 07 AM](https://user-images.githubusercontent.com/52669884/89801544-558be000-dafe-11ea-9804-6f40f0c97dc6.png)
![Screen Shot 2020-08-10 at 11 33 19 AM](https://user-images.githubusercontent.com/52669884/89801559-59b7fd80-dafe-11ea-9a8b-dba4d8ebaf4b.png)
![Screen Shot 2020-08-10 at 11 39 51 AM](https://user-images.githubusercontent.com/52669884/89801565-5d4b8480-dafe-11ea-8712-c8d004ebb705.png)
